docs/index.md

```diff
@@ -1,4 +1,4 @@
-![CI](https://github.com/mwouts/jupytext/actions/workflows/continuous-integration.yml/badge.svg?branch=main)
+![CI](https://github.com/mwouts/jupytext/actions/workflows/ci.yml/badge.svg?branch=main)
```

Comment:
Hopefully this one reports as passed. If not we can look at how to report only the las job in the CI

Reply:
Yes it does! ![CI](https://github.com/mwouts/jupytext/actions/workflows/ci.yml/badge.svg?branch=main)